### PR TITLE
merge linterns

### PR DESCRIPTION
**Pull request by sergiogval**
Merge linterns to main branch

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 node_modules
+=======
+node_modules
+>>>>>>> dc917884158d12cc891e6cdc865dcf02a81fe033
```
Comment (sergiogval) [inline .gitignore]: The best is to stick to the previous file in order to avoid conflicts.

File: index.html [hunk 2]
```diff
@@ -2,8 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content= "width=device-width, initial-scale=1.0">
-    <title>Trial project 1np</title>
```
Comment (sergiogval) [inline index.html]: Keep these tags, in order to avoid linterns errors.

File: index.html [hunk 3]
```diff
@@ -70,8 +69,8 @@ <h1 id="title">Survey Form</h1>
   </div>
```
Comment (sergiogval) [inline index.html]: Maybe the best is not change index.html file.

**Review by sergiogval — COMMENTED** [review]
Going we so far!

https://user-images.githubusercontent.com/67393791/129609754-6c65be80-546f-413d-9bf2-306d04cc2c7c.mp4



Just a few details that I menction bellow:

File: index.html (hunk 3 above)
Comment (mricanho) [reply index.html]: - [ ] Remember to always write the why you are requesting the changes, this is to clarify the concepts for the student

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @sergiogval, 

You are close to finishing this milestone, just a few changes and you are good to go! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Check my comments under your comments, you can edit your comments and ask for a new Code Review.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almostthere

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+    <title>Trial project 1np</title>
```
[the remark this answers is not in the document]
Comment (mricanho) [reply index.html]: - [ ] Try to be more polite, remember that sometimes the reading could create misunderstandings

File: index.html (hunk 3 above)
Comment (sergiogval) [reply index.html]: done.

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (sergiogval) [reply index.html]: you are right. I've edited all the comments in order to comply with it.

File: index.html (hunk 3 above)
Comment (henatan99) [reply index.html]: - This issue is well addressed! :clap:

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (henatan99) [reply index.html]: - Thanks to consider and make changes according to the reviewer's comment here. :+1:

**Review by henatan99 — APPROVED** [review]
Hi @sergiogval , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

- [ ] OPTIONAL: I think you can add review status in your review summary like `Changes Requested` or `Approved`. In addition, you could also have a summary of the review you have made in the inline comments. 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**